app/controllers/users/contact_details.js

```diff
@@ -0,0 +1,174 @@
+import Ember from "ember";
+import AjaxPromise from "stock/utils/ajax-promise";
+import AsyncMixin, { ERROR_STRATEGIES } from "stock/mixins/async";
+import ImageUploadMixin from "stock/mixins/image_upload";
+
+export default Ember.Controller.extend(AsyncMixin, ImageUploadMixin, {
+  user: Ember.computed.alias("model.user"),
+
+  titles: Ember.computed(function() {
+    let translation = this.get("i18n");
+    let mr = translation.t("order.user_title.mr");
+    let mrs = translation.t("order.user_title.mrs");
+    let miss = translation.t("order.user_title.miss");
+    let ms = translation.t("order.user_title.ms");
+
+    return [
+      { name: mr, id: "Mr" },
+      { name: mrs, id: "Mrs" },
+      { name: miss, id: "Miss" },
+      { name: ms, id: "Ms" }
+    ];
+  }),
```

Comment:
I smell code duplication here. This was used in addUser.js as well

Probably you can make a method in userService, which you can reuse in both